src/briefcase/integrations/android_sdk.py

```diff
@@ -557,7 +566,7 @@ def list_packages(self):
         try:
             # check_output always writes its output to debug
             self.tools.subprocess.check_output(
-                [os.fsdecode(self.sdkmanager_path), "--list_installed"],
```

Comment:
Removed `os.fsdecode()` calls for `subprocess` args since it has built-in `fspath` support.